Header header
time request_stamp
string hardware_id
NameValueFloat[] name_value_floats
NameValueInt[] name_value_ints
NameValueInt[] name_value_bools
NameValueString[] name_value_strings
NameValueString[] name_value_enums